pkg/scheduler/scheduler.go

```diff
@@ -342,6 +348,9 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 
 			// Run "postbind" plugins.
 			sched.framework.RunPostBindPlugins(bindingCycleCtx, sub, targetClusters)
+
+			// TODO: update subscription condition
+			sched.updateSubscriptionSchedulingSuccess(sub)
```

Comment:
Scheduler should only focuses on scheduling. Better not add this here.

Reply:
> Scheduler should only focuses on scheduling. Better not add this here.

It's updating the scheduling result (success or failure) to SubscriptionStatus condition, I have made a new commit about this. 

[[Scheduling: Update the scheduling result (success or failure) to SubscriptionStatus condition](url)](https://github.com/clusternet/clusternet/pull/350/commits/a27c1321ee23d4143402e65a5f0bc2ee5289a201)